app/Metadata/Extractor.php

```diff
@@ -96,15 +96,30 @@ public function extract(string $filename, string $type): array
 			}
 		}
 
+		// Attempt to get sidecar metadata if it exists, make sure to check 'real' path in case of symlinks
+		$sidecarData = [];
+
 		try {
 			// this can throw an exception in the case of Exiftool adapter!
 			$exif = $reader->read($filename);
+
+			$realFile = readlink($filename) ?: $filename;
+			if (Configs::hasExiftool() && file_exists("$realFile.xmp")) {
+				$sidecarData = $reader->read("$realFile.xmp")->getData();
```

Comment:
I don't really like the syntax `"$var.something"`, we tend to stick with `$var . ".something"` as this is slightly more readable (also syntax coloring wise).